README.md

```diff
@@ -238,6 +238,33 @@ a.b = 1
 a.b.c = 2
 ```
 
+Defining dotted keys out-of-order is valid currenctly, but discouraged:
```

Comment:
Maybe just "Defining dotted keys out-of-order is discouraged:" – That's it's valid is also pointed out at the top of the example, and "currenctly" (actually there's a typo in there) suggests that it will become invalid in the future – which might nor might not happen.

Reply:
I wrote "currently" because:
https://github.com/toml-lang/toml/issues/446#issuecomment-345363045
https://github.com/toml-lang/toml/issues/446#issuecomment-391778942

But I forgot this example, which is reasonable:
https://github.com/toml-lang/toml/issues/446#issuecomment-391861244

So I removed the "currently".